@wix/wnpm-ci not found - breaks npm install

Please update package.json. It seems that @wix/wnpm-ci has been moved into wnpm-ci in npm registry.
```
npm ERR! Linux 4.9.36-moby
npm ERR! argv "/usr/bin/nodejs" "/usr/bin/npm" "install"
npm ERR! node v6.11.2
npm ERR! npm  v3.10.10
npm ERR! code E404

npm ERR! 404 Not found : @wix/wnpm-ci
npm ERR! 404
npm ERR! 404  '@wix/wnpm-ci' is not in the npm registry.
npm ERR! 404 You should bug the author to publish it (or use the name yourself!)
npm ERR! 404 It was specified as a dependency of 'redux-saga-tester'
npm ERR! 404
npm ERR! 404 Note that you can also install from a
npm ERR! 404 tarball, folder, http url, or git url.
```

Even installing just redux-saga-tester this error exists:
```
> npm install redux-saga-tester

npm ERR! code E404
npm ERR! 404 Not Found: @wix/wnpm-ci@^6.1.0
```

--- a/package.json
+++ b/package.json
@@ -11,7 +11,7 @@
     "test-dev": "mocha --watch --compilers js:babel-register",
     "compile": "babel -d dist/ src/",
     "build": ":",
-    "release": "npm install @wix/wnpm-ci && wnpm-release -- --no-shrinkwrap"
+    "release": "npm install wnpm-ci && wnpm-release -- --no-shrinkwrap"
   },
   "repository": {
     "type": "git",

